core/src/db/authorize/constants.ts

```diff
@@ -0,0 +1,24 @@
+export enum Resource {
+  Franchise = 'franchise',
+  Game = 'game',
+  GameMode = 'game_mode',
+  Match = 'match',
+  Player = 'player',
+  PlayerStat = 'player_stat',
+  Role = 'role',
+  Round = 'round',
+  ScheduleGroup = 'schedule_group',
+  ScheduleGroupType = 'schedule_group_type',
+  SkillGroup = 'skill_group',
+  Team = 'team',
+  TeamStat = 'team_stat',
+  User = 'user',
+  UserAuthAccount = 'user_auth_account',
+}
+
+export enum ResourceAction {
+  Create = 'create',
+  Read = 'read',
+  Update = 'update',
+  Delete = 'delete',
```

Comment:
There's a section in `lib` that has items like this so that they can be shared between clients and the backend, might be good to put stuff there to ease interacting with it on FE

Reply:
Fixed.